libs/blocks/aside/aside.css

```diff
@@ -137,11 +177,48 @@
   color: transparent;
 }
 
-.aside.split .split-image img {
+.aside.split .split-image img,
+.aside.split .split-image video {
   position: relative;
   min-height: 270px;
 }
 
+.aside.split .split-image video {
+  display: block;
+}
+
+.aside.split .mobile-square img,
+.aside.split .mobile-square video {
+  aspect-ratio: var(--aspect-ratio-square);
+}
+
+.aside.split .mobile-wide img,
+.aside.split .mobile-wide video {
+  aspect-ratio: var(--aspect-ratio-wide);
+}
+
+.aside.split .mobile-standard img,
+.aside.split .mobile-standard video {
+  aspect-ratio: var(--aspect-ratio-standard);
+}
+
+.aside.split .icon-stack-area li img {
+  height: 40px;
```

Comment:
Are the icons in the stack area always a square? I've noticed the typical app icon has an `Intrinsic aspect ratio: | 40∶39`. Id say we do the same as adobe.com and only set the `width` and `max-width` and have height to inherit/auto. 


 size. Setting this hard value will cause some distortion on the image.

Reply:
Also shouldn't we be using our icon size variables?
```
 --icon-size-xxl: 80px;
  --icon-size-xl: 64px;
  --icon-size-l: 56px;
  --icon-size-m: 40px;
  --icon-size-s: 32px;
  --icon-size-xs: 24px;
  --icon-size-xxs: 16px;

Reply:
@ryanmparrish icon sizes were recently merged after this PR. I have updated the icon sizes as suggested to a standard icon-size variable name.